.eslintignore

```diff
@@ -1 +1,2 @@
 www/Promise.js
+*.spec.js
```

Comment:
Ignore the `spec` files as we can use fancy things such as const, let, and arrow functions in jest tests.